src/main/java/lotto/model/LotteryResults.java

```diff
@@ -30,11 +29,8 @@ public static void increase(Grade grade) {
 
     public static List<LotteryResult> getLotteryResult() {
         List<LotteryResult> lotteryResults = new ArrayList<>();
-        for (Entry<Grade, Integer> gradeIntegerEntry : array()) {
-            lotteryResults.add(
-                    new LotteryResult(gradeIntegerEntry.getKey(), gradeIntegerEntry.getValue()));
-        }
-
+        array().stream().filter(grade-> grade.getKey() != Grade.NONE).forEach(gradeIntegerEntry ->
+                lotteryResults.add(new LotteryResult(gradeIntegerEntry.getKey(), gradeIntegerEntry.getValue())));
         return lotteryResults;
```

Comment:
- 객체지향 생활 체조 원칙 중, `규칙 4: 한 줄에 점을 하나만 찍는다.`를 적용해보면 어떨까요?
- 다음처럼 작성해볼 수 있을 것 같아요.
```suggestion
        return gradeMap.keySet()
            .stream()
            .filter(grade -> grade != Grade.NONE)
            .map(grade -> new LotteryResult(grade, gradeMap.get(grade)))
            .collect(Collectors.toList());
```